src/components/directory-content.js

```diff
@@ -1087,15 +1115,17 @@ const DirectoryContent = () => {
                 onClose={handleCloseScriptContingency}
                 onError={handleError}
                 title={useIntl().formatMessage({ id: 'editContingencyList' })}
-                type={elementType.SCRIPT_CONTINGENCY_LIST}
+                type={elementType.CONTINGENCY_LIST}
+                subtype={contingencyListSubtype.SCRIPT}
```

Comment:
I don't think you need this subtype props here. We check the subtype before setting `openScriptContingencyDialog` then it's obviously `contingencyListSubtype.SCRIPT`

Reply:
You're right. Done.